solutions/bronze/usaco-1276.mdx

```diff
@@ -143,7 +143,7 @@ using namespace std;
  * Check if the given 'air_conditioners' satisfy the constraints of the cows
  */
 bool check(vector<array<int, 3>> &cows,
-		   vector<array<int, 4>> &air_conditioners) {
+           vector<array<int, 4>> &air_conditioners) {
```

Comment:
c'mon the ci removes my tabs and converts them to spaces??

Reply:
tabs for indentation, spaces for formatting
stupid, but hey, what u gon do about it